common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -10,6 +10,15 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
 
   val googleMapsUrl = "https://www.google.com/maps/embed/v1/place?center=-3.9834936%2C12.7024497&key=AIzaSyBctFF2JCjitURssT91Am-_ZWMzRaYBm4Q&zoom=5&q=Democratic+Republic+of+the+Congo"
 
+  val interactiveWithoutiFrameWrapper = """<figure class="element element-interactive interactive" data-interactive="https://foo.bar/foo/bar/boot.js" data-canonical-url="https://foo.bar/foo/bar/boot.js">
+                                          |<figcaption class="caption caption--img caption--main">
+                                          |Our comprehensive
+                                          |<a href="https://someinteractive.url/foo/ng-interactive/bar/foobar">interactive results page</a>: follow the results as they come in.
+                                          |</figcaption>
+                                          |</figure>"""
```

Comment:
Its a thing called "stripMargin" that is supposed to remove leading spaces from multi-line strings - see about 1/4 of the way down this page: http://alvinalexander.com/scala/scala-multiline-strings-heredoc-syntax
To be honest, I had to look that up - they were auto-inserted by intellij when I pasted text between the """'s.
Open to removing it if anyone thinks I should.